### ISSUE TITLE
FieldNode denormalization ignores 

### ISSUE THREAD

**Issue by jeroldalbertson-wf** [gql-dart/ferry#497]
👋 Hiyo, I'm running into an issue in https://pub.dev/packages/graphql (but probably more so in it's dependency https://pub.dev/packages/normalize) where the  acceptPartialData = false  value I'm providing to GraphQLCache is being ignored when I also configure it with custom FieldPolicies. It appears this acceptPartialData value is ignored when matching FieldNodes are denormalized https://github.com/gql-dart/ferry/blob/636d5756ca548d9b546623cd4e58d1247a53198a/packages/normalize/lib/src/policies/field_policy.dart#L52 . This is causing exceptions when I try to get parsedData from  cacheAndNetwork QueryResults (asking for a complete entity) when there is an incomplete cached entity available. Any help is much appreciated.

**Comment by knaeckeKami** [comment]
@smkhalsa do you maybe remember if there is a  reason why ` allowPartialData: true` is set to true there instead of  `_config.allowPartialData ` ?

**Comment by smkhalsa** [comment]
I don't. It's been quite a while. Sorry.

**Comment by brianblanchard-wf** [comment]
@knaeckeKami would you be open to allowing `allowPartialData` to be set via an optional named param on `readField`? 

A default value of `true` could be provided for this new param to limit any unintended side effects.

**Comment by knaeckeKami** [comment]
yes, that would be a safe way of letting users control this without breaking anything
